core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -443,6 +431,7 @@ private ExecutionModel parseCopy(GenericLexer lexer) throws SqlException {
     }
 
     private ExecutionModel parseCreateStatement(GenericLexer lexer, SqlExecutionContext executionContext) throws SqlException {
+        // TODO: if not 'table' it should pass to parseUnexpected()
```

Comment:
nit: `expectTok` already throws when the token is different from `table`. Why do we need this TODO?

Reply:
this is an unfinished sql parser extension API, we will remove or act on it later